meson_cpu/ppc64/meson.build

```diff
@@ -0,0 +1,38 @@
+source_root = meson.project_source_root()
+mod_features = import('features')
+compiler_id = meson.get_compiler('c').get_id()
+
+VSX = mod_features.new(
+  'VSX', 1, args: '-mvsx',
+  test_code: files(source_root + '/numpy/distutils/checks/cpu_vsx.c')[0],
```

Comment:
Maybe for later: I think we can move all these files from `numpy/distutils/checks/` to `meson_cpu/checks/`?

Reply:
I kept it for later till we drops the support of distutils, since I want to refactor theses tests under each subdir of each architecture rather than having them under single directory, for example:
- `meson_cpu/x86/checks`
- `meson_cpu/arm/checks`